### ISSUE TITLE
[BITV]: Add extra aria labels to several sections

### ISSUE THREAD

**Issue by emoral435** [nextcloud/server#42855]
From #42744, [these] are the places that need aria-labels on. This wil take several different PR's to fix, as some fixes will require libraries being changed, and nextcloud/server being changed as well!

![Image](https://github.com/nextcloud/server/assets/110193237/9c398bb1-900b-48ed-8902-fef9caf10153)

**Comment by szaimen** [comment]
Not sure if this implementation is correct in `settings/user/sharing`:
![image](https://github.com/nextcloud/server/assets/42591237/e58ce992-98b5-4bd0-b794-e73629d8965e)
also here:
![image](https://github.com/nextcloud/server/assets/42591237/c8c450de-ba77-4567-9921-7e838a83b63a)
and all the other sides...

**Comment by emoral435** [comment]
My bad - once https://github.com/nextcloud/server/pull/43064 gets merged, good to be checked 👍

**Comment by szaimen** [comment]
I checked all mentnioned pages and found two missing places:

![image](https://github.com/nextcloud/server/assets/42591237/8bc59500-7024-49af-adf4-c9a910f5e547)

![image](https://github.com/nextcloud/server/assets/42591237/f5f2ff8f-cfbb-410d-a284-328bbf43ae48)
